src/main/java/com/blogify/blogapi/model/Comment.java

```diff
@@ -0,0 +1,42 @@
+package com.blogify.blogapi.model;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "comment")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+public class Comment {
+    @Id
+    private String id;
+
+    @Column(name = "content")
```

Comment:
not need to spesify @Column(name = "content") when 
- the entity have the same name (ex: content = content)
- or if the entity have right conversion  (ex: creation_date_time = creationDateTime) all _ change tu upercaset 
so delete @Column(name = "content")